addon/mixins/list-input.js

```diff
@@ -0,0 +1,71 @@
+// import _ from 'lodash'
+import Ember from 'ember'
+// import computed, {readOnly} from 'ember-computed-decorators'
+
+import * as utils from '../components/utils'
+
+export default Ember.Mixin.create({
+  init () {
+    this._super(...arguments)
```

Comment:
Do you get to call `this._super()` in a mixin? What does that even mean, since it doesn't participate in the inheritance chain?